package.json

```diff
@@ -3,31 +3,41 @@
   "version": "2.0.0",
   "private": true,
   "devDependencies": {
+    "@kentico/kontent-model-generator": "^4.1.0",
+    "@simply007org/kontent-react-components": "^0.0.8",
+    "@types/lodash": "^4.14.181",
+    "@types/react-helmet": "^6.1.5",
+    "@types/validator": "^13.7.2",
+    "@types/webpack-env": "^1.16.3",
     "cpy-cli": "^4.0.0",
     "cross-env": "^7.0.2",
-    "dateformat": "^3.0.3",
     "husky": "^4.2.5",
     "lint-staged": "^10.2.7",
     "prettier": "^2.0.5",
     "react-scripts": "^5.0.0"
   },
   "dependencies": {
     "@kentico/kontent-delivery": "11.7.0",
-    "@simply007org/kontent-react-components": "0.0.6",
+    "@simply007org/kontent-react-components": "0.0.8",
     "@simply007org/react-spinners": "0.0.3",
-    "google-maps-react": "^2.0.6",
-    "immutable": "^4.0.0-rc.12",
-    "lodash": "^4.17.15",
+    "@testing-library/jest-dom": "^5.16.3",
+    "@testing-library/react": "^12.1.4",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.4.1",
+    "@types/node": "^16.11.26",
+    "@types/react": "^17.0.43",
+    "@types/react-dom": "^17.0.14",
+    "lodash": "^4.17.21",
     "qs": "^6.9.4",
-    "react": "^18.0.0",
+    "react": "^17.0.2",
     "react-cookie": "^4.1.1",
-    "react-dom": "^18.0.0",
-    "react-helmet": "^6.1.0",
-    "react-router-dom": "^5.2.0",
-    "react-scroll": "^1.7.16",
-    "react-translate": "^7.0.1",
-    "universal-cookie": "^4.0.3",
-    "validator": "^13.1.1"
+    "react-dom": "^17.0.2",
+    "react-helmet": "^6.0.0",
+    "react-intl": "^5.24.8",
+    "react-router-dom": "^6.2.2",
+    "universal-cookie": "^4.0.4",
```

Comment:
Can we reduce 2 cookie libraries (universal-cooke vs. react-cookie) to one?

Reply:
[334dfdd](https://github.com/Kentico/kontent-sample-app-react/pull/206/commits/334dfdd3ecf49b6bea32c78f44043046d8d6abb9)